src/components/view/Garage/ControlPanel/controlPanel.ts

```diff
@@ -0,0 +1,72 @@
+import './control_panel.css';
+
+class ControlPanel {
+    constructor() {
+        const app = document.getElementById('app');
```

Comment:
You are using same methods multiple times to create every element of the interface. It's a good idea to create class that will acceps certain control parameters and create this contol.